checkov/github_actions/image_referencer/provider.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+from typing import Any
+
+from checkov.common.images.image_referencer import Image
+
+
+class GithubActionProvider:
+    __slots__ = ("workflow_config", "file_path", "workflow_line_numbers")
+
+    def __init__(self, workflow_config: dict[str, Any], file_path: str, workflow_line_numbers: list[tuple[int, str]]):
+        self.workflow_config = workflow_config
+        self.file_path = file_path
+        self.workflow_line_numbers = workflow_line_numbers
+
+    @staticmethod
+    def _get_start_end_lines(entity: dict[str, Any]) -> tuple[int, int]:
+        return entity.get('__startline__', 0), entity.get('__endline__', 0)
```

Comment:
can use the consts from `from checkov.common.util.consts import START_LINE, END_LINE`